common/v2/services/WalletService/walletService.ts

```diff
@@ -27,6 +27,7 @@ import { unlockWeb3 } from './web3';
 export const WalletFactory = (walletId: WalletId): WalletService | any => {
   switch (walletId) {
     case WalletId.METAMASK:
+    case WalletId.TRUST:
```

Comment:
It's strange that we don't add the case for the other web3 wallets here as well.